.python-versions

```diff
@@ -4,3 +4,5 @@
 3.9.18
 3.8.18
 3.8.12
+pypy@3.10.14
+pypy@3.9.19
```

Comment:
Maybe just use `uv python install` for these in the PyPy integration jobs we have in CI instead? Are you going to make use of these in unit tests?

Reply:
Thanks for pointing out, I wasn't aware of that.